eventuate-local-java-common-util/src/main/java/io/eventuate/local/java/common/util/ZkLeaderSelector.java

```diff
@@ -0,0 +1,84 @@
+package io.eventuate.local.java.common.util;
+
+import org.apache.curator.framework.CuratorFramework;
+import org.apache.curator.framework.recipes.leader.CancelLeadershipException;
+import org.apache.curator.framework.recipes.leader.LeaderSelector;
+import org.apache.curator.framework.recipes.leader.LeaderSelectorListener;
+import org.apache.curator.framework.state.ConnectionState;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.UUID;
+
+public class ZkLeaderSelector implements CommonLeaderSelector {
```

Comment:
This should be in leadership.zookeeper package.
And a corresponding ...leadership-zookeeper module

Reply:
will do